conf/manager.env

```diff
@@ -8,3 +8,4 @@ RBAC_URL=http://platform_mock:8000
 API_VERSION=v1
 STATUS_TIMEOUT=10
 CYNDI_ENABLED=TRUE
+REQUEST_RETRIES=10
```

Comment:
REQUEST_RETRIES is in manager.env and in vmaas-sync.env, if the variable is in more than 1 file, it should be put in common.env